egui/src/widgets/color_picker.rs

```diff
@@ -206,14 +206,15 @@ pub enum Alpha {
 
 fn color_text_ui(ui: &mut Ui, color: impl Into<Color32>) {
     let color = color.into();
+    let button_text = &ui.ctx().localization().click_copy.clone();
```

Comment:
```suggestion
    let button_text = ui.ctx().localization().click_copy;
```

Should work, no? Since `click_copy` is `&'static str` this should just copy that.

Reply:
That's true. I've removed all the clones and everything still works.